docs/asciidoc/back_matter.adoc

```diff
@@ -39,3 +37,94 @@ This spec defines a standard **Data Repository Service (DRS) API** (“the yello
+
+  "urlPattern" : "https://drs.myexample.org/ga4gh/drs/v1/objects/{$id}"
+
+=== Calling the n2t.net API as a Client
+
+It takes one API call to get the URL pattern.
+
+The client makes a GET request to n2t.net to find information about the namespace. (Note the trailing colon.)
+
+  GET https://n2t.net/drs.42:
+
+This request returns a text structure including a redirect field, whose value is an URL pattern containing a `$id` parameter, such as:
+
+   redirect: https://drs.myexample.org/ga4gh/drs/v1/objects/$id
+
+=== Caching with Compact Identifiers
+
+Identifiers.org/n2t.net compact identifier resolver records do not change frequently.  This reality is  useful for caching resolver records and their URL patterns for performance reasons.  Builders of systems that use compact identifier-based DRS URIs should cache prefix resolver records from identifiers.org/n2t.net and occasionally refresh the records (such as every 24 hours).  This approach will reduce the burden on these community services since we anticipate many DRS URIs will be regularly resolved in workflow systems.  The implementation of the caching approach is up to system builders but we recommend at least caching the mapping of prefixes to URL patterns. System builders may also choose to cache the compact identifier accession to final DRS URL records as well, but that decision/design is at the discretion of the system builder.  Alternatively, system builders may decide to directly mirror the registries themselves, instructions are provided on the identifiers.org/n2t.net websites.
+
+=== Security with Compact Identifiers
+
+As mentioned earlier, identifiers.org/n2t.net performs some basic verification of new prefixes and provider code mirror registrations on their sites.  However, system builders may have certain security compliance requirements and regulations that prohibit relying on an external site for resolving compact identifiers.  In this case, systems under these security and compliance constraints may wish to whitelist certain compact identifier resolvers and/or vet records from identifiers.org/n2t.net before enabling in their system.
+
+=== Accession Encoding to Valid DRS IDs
+
+The compact identifier format used by identifiers.org/n2t.net does not percent-encode reserved URI characters but, instead, relies on the first ":" character to separate prefix from accession. Since these accessions can contain any characters, and characters like "/" will interfere with DRS API calls, you _must_ percent encode the accessions extracted from DRS compact identifier-based URIs when using as DRS IDs in subsequent DRS GET requests.  An easy way for a DRS client to handle this is to get the initial DRS object JSON response from whatever redirects the compact identifier resolves to, then look for the `self_uri` in the JSON, which will give you the correctly percent-encoded DRS ID for subsequent DRS API calls such as the `access` method.
```

Comment:
I'm confused by this guidance -- given that DRS URIs are already required to be URI-friendly, there shouldn't be anything left to encode. Unless I'm missing something, let's delete this section.

Reply:
Identifiers.org/n2t.net accessions do not use encoding while DRS IDs (and any calls made to DRS API endpoints) do.  Hence this description.  We can remove it but it will leave some key behavior underspecified and will likely cause confusion by implementers.  I strongly recommend leaving this and discussing with our drivers/implementers to clarify language.